refactor(sandboxes): update package.json for each sample to use latest source

dysfunc opened the pull request:
I made the change that produces the smallest diff. You could also create an alias inside vite, but this does the same thing.

```js
import { defineConfig } from 'vite';
import { svelte } from '@sveltejs/vite-plugin-svelte';
import path from 'path';

export default defineConfig({
  plugins: [svelte()],
  resolve: {
    alias: {
      'svelte-chartjs': path.resolve(__dirname, '../../')
    }
  }
});
```

niemyjski commented:
Do you know if this will work from code sandbox https://codesandbox.io/s/github/SauravKanchan/svelte-chartjs/tree/master/sandboxes/bar ?

niemyjski commented on sandboxes/bar/package.json:
```diff
@@ -6,7 +6,6 @@
   },
   "dependencies": {
     "svelte": "^4.0.0",
-    "svelte-chartjs": "^3.0.0",
```
Should this be? "svelte-chartjs": "../../",

dysfunc commented:
> Do you know if this will work from code sandbox https://codesandbox.io/s/github/SauravKanchan/svelte-chartjs/tree/master/sandboxes/bar ?

Oh, if you're going to link out then those files shouldn't change. Once you deploy the new version, that warning will go away about the missing exports. I could also try linking to the master branch

dysfunc commented:
Are you planning to treat this as a major release (`v4.0.0`) since we modified bundling and updated the repo? If so, we could change the sandbox to reference `^4.0.0` now. Either way, these changes are no good since it requires installing the specific chartjs version.

niemyjski commented:
@dysfunc good catch, I didn't think so because it was just the bundling folder which tools should pick up on. Is there anything I'm missing?

dysfunc commented:
I think it's fine. It's just a way to control who receives the latest since they'll need to manually update the major, but I don't think that's needed.
